### ISSUE TITLE
Unable to Reproduce Model Results - Pearson Correlation Discrepancy

### ISSUE THREAD

**Issue by TimothyChen225** [Tsinghua-gongjing/StructureImpute#5]
```bash
#!/bin/bash
work_space=$(dirname $0)
outdir=$work_space/out_batch_800_lr_00001_stop100
if [ ! -d $outdir ] ;then 
    mkdir $outdir
fi

echo $outdir
# time CUDA_VISIBLE_DEVICES=$1 
name=$(basename $work_space)
python -u -d tools/main.py \
    --batch_size 800 \
    --test_batch_size 720 \
    --lr 0.0001 \
    --train_type trainHasNull_lossAll \
    --monitor_val_loss train_hasnull_validate_onlynull \
    --filename_train ./data/processed/icSHAPE_hek293_vivo.train \
    --filename_validation ./data/processed/icSHAPE_hek293_vivo.val \
    --filename_prediction $outdir/prediction.txt \
    --logdir $outdir/tfb \
    |tee -a $outdir/log.txt
```
Hello, I trained the model using the above code, and then passed the obtained prediction.txt along with /data/structure_score/hek_wc_vivo/validation.txt to compare_true_and_predict.py for Pearson correlation calculation. The resulting Pearson coefficient is 0.54, which shows a significant difference from the 0.61 reported in the paper (Figure 2b). The parameters used were: Early stopping of 100, learning rate 0.0001, and the other parameters are the default settings in main.py. I would like to inquire: where might the problem lie? Thank you.
